tests/monit/test_monit_status.py

```diff
@@ -5,18 +5,45 @@
 
 import pytest
 
+from tests.common.utilities import wait_until
 from tests.common.helpers.assertions import pytest_assert
 
 logger = logging.getLogger(__name__)
 
 pytestmark = [
-    pytest.mark.topology('any')
+    pytest.mark.topology('any'),
+    pytest.mark.disable_loganalyzer
 ]
+@pytest.fixture
+def disable_lldp(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    duthost.command("docker exec lldp supervisorctl stop lldpmgrd")
+    if duthost.is_multi_asic:
+        duthost.command("docker exec lldp0 supervisorctl stop lldpmgrd")
+    yield
+    duthost.command("docker exec lldp supervisorctl start lldpmgrd")
+    if duthost.is_multi_asic:
+        duthost.command("docker exec lldp0 supervisorctl start lldpmgrd")
 
+def check_monit_last_output(duthost):
+    monit_status_result = duthost.shell("sudo monit status \'lldp|lldpmgrd\'", module_ignore_errors=True)['stdout_lines']
+    indices = [i for i, s in enumerate(monit_status_result) if 'last output' in s]
+    monit_last_output = monit_status_result[indices[0]]
+    if duthost.is_multi_asic:
+        return "'python /usr/bin/lldpmgrd' is not running in host and in namespace asic0" in monit_last_output
+    else:
+        return "'python /usr/bin/lldpmgrd' is not running in host" in monit_last_output
 
 def test_monit_status(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
     duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
     monit_status_result = duthost.shell("sudo monit status", module_ignore_errors=True)
 
     exit_code = monit_status_result["rc"]
     pytest_assert(exit_code == 0, "Monit is either not running or not configured correctly")
+
+def test_monit_reporting_message(duthosts, enum_rand_one_per_hwsku_frontend_hostname, disable_lldp):
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    if '201911' not in duthost.os_version:
+        pytest.skip("Skip test as monit might not being used for process monitoring.")
```

Comment:
@yozhao101 is this valid for master also ? Can you please confirm.

Reply:
You mean in master branch, whether we still use Monit to monitor the running status of critical processes, right?

Reply:
@yozhao101 that is the qustion

Reply:
I am wondering whether we need create a separate pytest script for this test case since the current script was used to test the status of Monit?

Reply:
I think we can keep this under the umbrella of monit. If need we can rename test as `test_monit.py`. However can you respond to the question is this case applicable for `master` also ?

Reply:
Yes, I think the master branch still use Monit to monitor the critical processes.

Reply:
thanks i will remove the image version check then.

Reply:
remove `201911 image`  check